### PR TITLE
add noquotes flag and use quotes by condition in dump method

### PR DESCRIPTION
**Pull request by vaivanov**

### CODE REVIEW

**Comment by donbeave** [comment]
@dtan4 any way to merge this PR?

**Comment by donbeave** [comment]
@dtan4
Any hope to merge or review these changes?

File: cmd/dump.go [hunk 1]
```diff
@@ -70,7 +81,11 @@ func doDump(cmd *cobra.Command, args []string) error {
 
 		for _, secret := range secrets.Items {
 			for key, value := range secret.Data {
-				lines = append(lines, key+"="+strconv.Quote(string(value)))
+				line := string(value)
+				if !dumpOpts.noquotes {
+					line = strconv.Quote(line)
+				}
+				lines = append(lines, key+"="+line)
```
Comment (dtan4) [inline cmd/dump.go]:
```suggestion
				v := string(value)
				if !dumpOpts.noquotes {
					v = strconv.Quote(v)
				}
				lines = append(lines, key+"="+v)
```

**Comment by vaivanov** [comment]
Hello, this PR is also updated after your review. Thanks!

**Pull request merged**